src/pages/api/applications/application.ts

```diff
@@ -0,0 +1,39 @@
+import { Types } from "mongoose";
+import {
+  createNonprofitApplication,
+  getApplication,
+} from "server/mongodb/actions/Applications";
+import APIWrapper from "server/utils/APIWrapper";
+import { NonprofitApplicationCreate } from "src/utils/types";
+
+export default APIWrapper({
```

Comment:
These endpoint handlers are correct, but for consistency consider the following changes for how they are organized:

Instead of application.ts, add index.ts and [projectId]/index.ts.  This would correspond to the endpoints /api/applications and /api/applications/someProjectId.  The POST request would be in index.ts, and the GET request would be in [projectId]/index.ts.  This may seem nitpicky right now, but remember that we refactored the routes so that we do not need to use actions with if statements to differentiate requests.  We wouldn't want to get a specific application in the same place as creating an application because when we need to get all applications (which is currently a sprint task), then we are right back to using if statements.  It also makes it easier to secure endpoints based on roles.  That was probably confusing so let me know if I need to explain further.